.goreleaser.yaml

```diff
@@ -32,9 +32,24 @@ changelog:
   use: github
 
 dockers:
+  - build_flag_templates:
+      - "--platform=linux/amd64"
+    image_templates:
+      - "ghcr.io/persona-id/{{.ProjectName}}:{{ .Tag }}-amd64"
+    goarch: arm64
+    skip_push: false
```

Comment:
I wonder if you set `skip_push: true` and as long as the `image_template` combines them you can push a single multi-arch tag? Might be cleaner?

Reply:
The `skip_push` option seems to be related to not pushing tags that look like release candidates - I think the tag with the arch is just a requirement of how it references them but it'd be nice to not have them and just have the single tag.

Reply:
Bummer, I see notices on the `docker_manifests` docs that state the image needs to be pushed.